### PR TITLE
Feature/add thirty five up

### PR DESCRIPTION
**Pull request by SpiderDan98**

### CODE REVIEW

File: bundles/thirty-five-up/LICENSE [hunk 1]
```diff
@@ -0,0 +1,9 @@
+The MIT License (MIT)
+
+Copyright (c) 2020 FOND OF GmbH
```
Comment (daniel-rose) [inline bundles/thirty-five-up/LICENSE:3]: 2021 instead of 2020

File: bundles/thirty-five-up/composer.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  "description": "Spryker Thirty Five Up Implementation Module.",
+  "license": "MIT",
+  "authors": [
+    {
+      "name": "Julian Zimmermann",
+      "email": "info@julianzimmermann.dev"
+    }
+  ],
+  "require": {
+    "php": ">=7.1"
```
Comment (daniel-rose) [inline bundles/thirty-five-up/composer.json:12]: 7.1 EOL. Maybe `>=7.3` will be better.

File: bundles/thirty-five-up/composer.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  },
```
Comment (daniel-rose) [inline bundles/thirty-five-up/composer.json:13]: Missing dependencies like spryker/sales-extension ...

**Review by daniel-rose — APPROVED** [review]
Please check replace section in main composer.json

**Review by gengjozsef — APPROVED** [review]

**Pull request merged**